singa_auto/model/dev.py

```diff
@@ -39,7 +40,7 @@ def tune_model(
         py_model_class: Type[BaseModel],
         train_dataset_path: str,
         val_dataset_path: str,
-        annotation_dataset_path: str,
+        task: str,
```

Comment:
when you change the API, you have to make sure that the new API works for old applications(models and datasets).
Is there any modle/dataset using annotation_dataset_path?

Reply:
'annotaion_dataset_path' was created by Naili. I just talked to him and we agreed to add 'if annotation_dataset_path:' as to make this line suitable for various models.

Reply:
there is a syntax error "default-arguments-follow-default-argument", because the default argument must be put as the last argument, while in your code task is not the default argument but placed at the final one
https://stackoverflow.com/questions/16932825/why-cant-non-default-arguments-follow-default-arguments